sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -925,6 +925,70 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     }
   }
 
+  /**
+   * Batch-delete messages.
+   *
+   * @param messageCount - number of messages to delete in a batch.
```

Comment:
nitpick: perhaps we can make it concise yet still clear?
```suggestion
   * @param messageCount - number of messages to delete.
```